manifests/plugin/salt/params.pp

```diff
@@ -6,6 +6,7 @@
   $enabled       = true
   $listen_on     = 'https'
   $autosign_file = "${foreman_proxy::params::etc}/salt/autosign.conf"
+  $autosign_key_file = "$/var/lib/foreman-proxy/salt/grains/autosign_key"
```

Comment:
Isn't there a variable similar to the line before (${foreman_proxy::params::etc)

Reply:
It's not. Other plugins like OpenSCAP and ShellHooks define the path similar as you see [here](https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/plugin/openscap.pp#L43) and [here](https://github.com/theforeman/puppet-foreman_proxy/blob/master/manifests/plugin/shellhooks.pp#L21).

Nevertheless, one could think of creating a variable in `./manifests/config.pp` or `./manifests/init.pp`.